pkg/util/patroni/patroni.go

```diff
@@ -78,3 +112,25 @@ func (p *Patroni) Failover(master *v1.Pod, candidate string) error {
 
 	return nil
 }
+
+// Status returns patroni status
+func (p *Patroni) Status(pod *v1.Pod) (*Status, error) {
```

Comment:
I don't see where this function is used?

Reply:
no

Reply:
why does it need to be part of this PR then?